### PR TITLE
Fix readability issues in Jupyter Notebooks with Markdown

### PR DESCRIPTION
**Pull request by minhduc0711**
I fixed a background color issue and made Markdown text in Jupyter notebooks more readable.
[Example notebook with markdown](https://github.com/jupyter/notebook/blob/master/docs/source/examples/Notebook/Working%20With%20Markdown%20Cells.ipynb)

One thing I was not able to do was making the table borders more visible:
image

image

My CSS knowlegdge is very limited so sorry in advance if I did something wrong :sweat_smile:

### CODE REVIEW

File: material-github.user.css [hunk 1]
```diff
@@ -1303,12 +1303,15 @@
  * | ==========================================
  */
 @-moz-document domain("render.githubusercontent.com"){
-	body[data-render-url="https://render.githubusercontent.com"]{
```
Comment (CharlieEtienne) [inline material-github.user.css:1306]: Why did you remove this part? I think you should just add `, #notebook`.

File: material-github.user.css [hunk 2]
```diff
@@ -1334,7 +1337,8 @@
 		color: var(--material-code-yellow);
 	}
 	.highlight .nf,
-	.highlight .nb{
```
Comment (CharlieEtienne) [inline material-github.user.css:1337]: Shouldn't `.nb` be red ?

File: material-github.user.css (hunk 1 above)
Comment (minhduc0711) [reply material-github.user.css:1306]: I dunno, I tried changing the background color for the body tag, but somehow nothing changed so I just replaced it with `#notebook`.

File: material-github.user.css (hunk 2 above)
Comment (minhduc0711) [reply material-github.user.css:1337]: I was trying to improve the colors of this latex code in the [markdown notebook](https://github.com/jupyter/notebook/blob/master/docs/source/examples/Notebook/Working%20With%20Markdown%20Cells.ipynb)
image
If it conflicts with your color convention, I can change it back.

Comment (CharlieEtienne) [reply material-github.user.css:1337]: Yes, it would affect all languages so I prefer you change it back.

File: material-github.user.css (hunk 1 above)
Comment (CharlieEtienne) [reply material-github.user.css:1306]: please use `body[data-render-url="https://render.githubusercontent.com"], #notebook` instead, since it would break other pages if you remove the `body[...` part

File: material-github.user.css [hunk 3]
```diff
@@ -1303,12 +1303,15 @@
  * | ==========================================
  */
 @-moz-document domain("render.githubusercontent.com"){
-	body[data-render-url="https://render.githubusercontent.com"]{
+	#notebook {
```
Comment (minhduc0711) [inline material-github.user.css]:
```suggestion
	body[data-render-url="https://render.githubusercontent.com"], #notebook {
```

File: material-github.user.css [hunk 4]
```diff
@@ -1334,7 +1337,8 @@
 		color: var(--material-code-yellow);
 	}
 	.highlight .nf,
-	.highlight .nb{
+	.highlight .sb,
```
Comment (minhduc0711) [inline material-github.user.css:1341]:
```suggestion
	.highlight .nb,
	.highlight .sb,
```

File: material-github.user.css [hunk 5]
```diff
@@ -1345,19 +1349,25 @@
 		color: var(--material-code-orange);
 	}
 	.highlight .s2,
-	.highlight .s1{
+	.highlight .s1,
+	.highlight .nb{
```
Comment (minhduc0711) [inline material-github.user.css]:
```suggestion
	.highlight .s1{
```

**Review by CharlieEtienne — APPROVED** [review]

**Comment by CharlieEtienne** [comment]
It's merged! Thanks @minhduc0711!
